packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -44,6 +43,9 @@ const useStyles = makeStyles(theme => ({
     minWidth: 260,
     maxWidth: 400,
   },
+  title: {
+    fontWeight: 500,
```

Comment:
Is this absolutely needed? Could we use another typography variant instead? It's a bit unfortunate to have overrides like these as it limits peoples ability to provide custom theming.